src/Fixer/Whitespace/NoExtraBlankLinesFixer.php

```diff
@@ -80,6 +81,10 @@ final class NoExtraBlankLinesFixer extends AbstractFixer implements Configurable
      */
     public function configure(array $configuration): void
     {
+        if (isset($configuration['tokens']) && \in_array('use_trait', $configuration['tokens'], true)) {
```

Comment:
do we usually do deprecations in patch releases, or reserve that for minors?

Reply:
good point, let's move this to 3.1